### PR TITLE
Move Accessing Clusters topic to Concepts.

### PR DESCRIPTION
**Pull request by iphutch**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2875)

### CODE REVIEW

**Comment by iphutch** [comment]
@steveperry-53 Please review

File: _data/concepts.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ toc:
   - docs/concepts/cluster-administration/guaranteed-scheduling-critical-addon-pods.md
   - docs/concepts/cluster-administration/static-pod.md
   - docs/concepts/cluster-administration/sysctl-cluster.md
+  - /docs/concepts/cluster-administration/access-cluster.md
```
Comment (steveperry-53) [inline _data/concepts.yml]: No forward slash before docs/concepts/...
It looks like this is causing the topic not to show up in the table of contents.

Comment (iphutch) [inline _data/concepts.yml]: Done

**Comment by steveperry-53** [comment]
@iphutch, Thanks for doing this; it looks good.

**Comment by chenopis** [comment]
@iphutch Welcome aboard, and thanks!

**Pull request merged**